aten/src/ATen/core/jit_type.h

```diff
@@ -547,7 +571,7 @@ struct CAFFE2_API TensorType : public Type {
         sizes_(tensor.sizes().size()),
         strides_(tensor.sizes().size()),
         requires_grad_(tensor.requires_grad()) {
-          if (!tensor.is_mkldnn()) {
+          if (!tensor.is_mkldnn() && !tensor.is_sparse()) {
```

Comment:
what's the reason for this change?

Reply:
we can't ask for strides on tensors that do not have them.  We now have tests that ask for the type of tuples, some of which contain sparse tensors, which uncovered this bug.